packages/gqltest/gqltest.js

```diff
@@ -61,10 +61,23 @@ class GQLResponse {
 // Asserts that the GraphQL response has status 200 and no errors.
 GQLResponse.prototype.expectOK = function () {
   chai.expect(this.response.status).to.equal(200);
-  chai.assert.notGraphQLError(this.body);
   return this;
 };
 
+
+// defaultTimeout can be used in describe.timeout()
+// for requests against a graphql endpoint.
+// Match stepzen AWS 60 second timeout.
+const defaultTimeout = 60000
```

Comment:
Probably ok for now, but `gqltest` was trying to stay GraphQL neutral (hence `stepzen.js`). They may need to be some thought on how to handle default timeouts nicely.